docker-for-windows/install.md

```diff
@@ -75,41 +72,35 @@ Looking for information on using Windows containers?
     It typically downloads to your `Downloads` folder, or you can run it from
     the recent downloads bar at the bottom of your web browser.
 
-2. Follow the install wizard to accept the license, authorize the installer, and
-   proceed with the install.
+2. Follow the instructions on the install wizard to accept the license, authorize the installer, and proceed with the install.
 
-    You are asked to authorize `Docker.app` with your system password during the
+    When prompted, authorize `Docker.app` with your system password during the
```

Comment:
This looks like something from the mac documention `Docker.app`, should it be `authorize the Docker Desktop Installer with` or something similar?

Reply:
Updated.